lib/protoparser/graphite/parser_test.go

```diff
@@ -19,6 +19,7 @@ func TestUnmarshalMetricAndTagsFailure(t *testing.T) {
 }
 
 func TestUnmarshalMetricAndTagsSuccess(t *testing.T) {
+	*sanitizeMetricName = true
```

Comment:
nit: it would be great to save original value of the flag and restore it after the test completion

Reply:
done